src/redux/actions/ForgotPasswordAction.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import * as ActionTypes from "../action_types";
+
+/* request to change password
```

Comment:
@kimotho-njoki Should this be above the function? I thought comments should be inside the function? @meshack-mbuvi

Reply:
Look at this file https://react-styleguidist.js.org/docs/documenting.html at @PAbishai

Reply:
Above!